### PR TITLE
[ADD] business_requirement_deliverable_resource_template_categ

### PR DESCRIPTION
**Pull request by martintorresvm**
- Extend Resources Templates with Task Category Main feature

### CODE REVIEW

**Comment by martintorresvm** [comment]
@elicoidal @seb-elico Please have a look

File: business_requirement_deliverable_resource_template_categ/README.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+.. figure:: static/img/bus_req_default.png
+   :width: 600 px
+   :alt: Set up your default resources lines.
+
+
+Usage
+=====
+
+#. Prepare your deliverables in the Product menu and add the expected RL
+
```
Comment (elicoidal) [inline business_requirement_deliverable_resource_template_categ/README.rst]: remove blank lines in the list

File: business_requirement_deliverable_resource_template_categ/README.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   :alt: The default resource lines are automatically added to your BR
+
+.. figure:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/222/8.0
+
+Known issues / Roadmap
+======================
+
+* Currently prices are not updated when the resource lines are uploaded in the
```
Comment (elicoidal) [inline business_requirement_deliverable_resource_template_categ/README.rst]: remove as this is fixed now

File: business_requirement_deliverable_resource_template_categ/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Deliverable Resource Template Category',
+    'category': 'Business Requirements Management',
+    'summary': """Manage default resource lines in your
```
Comment (elicoidal) [inline business_requirement_deliverable_resource_template_categ/__openerp__.py]: Manage default resource lines categories in your
deliverable templates

**Review by elicoidal — APPROVED** [review]
some details
@seb-elico

File: business_requirement_deliverable_resource_template_categ/views/business_requirement_resource_template.xml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    <data>
+        <record id="product_template_only_form_view" model="ir.ui.view">
+            <field name="name">product.template.form</field>
+            <field name="model">product.template</field>
+            <field name="inherit_id"
+                   ref="business_requirement_deliverable_resource_template.product_template_only_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='resource_lines']/tree[@editable='top']/field[@name='name']"
+                       position="after">
+                       <field name="categ_id"/>
```
Comment (seb-elico) [inline business_requirement_deliverable_resource_template_categ/views/business_requirement_resource_template.xml]: Wrong indent

**Review by seb-elico — CHANGES_REQUESTED** [review]
Almost LGTM

**Review by seb-elico — APPROVED** [review]

**Comment by elicoidal** [comment]
Fast tracking

**Pull request merged**